src/main/java/org/usfirst/frc4048/common/swervev3/vision/PoseDivation.java

```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc4048.common.swervev3.vision;
+
+import edu.wpi.first.math.Vector;
+import edu.wpi.first.math.numbers.N3;
+
+public class PoseDivation {
```

Comment:
What's this for? Did you mean `deviation`?

Reply:
yes, and I will add docs

Reply:
done